### PR TITLE
ssh-audit: update to 2.1.1

### PR DESCRIPTION
**Pull request by RunningDroid**
The original maintainer disappeared, this fork continues development.
Issue on original repo asking if the project is still active: arthepsy/ssh-audit#42

@abenson

### CODE REVIEW

**Comment by abenson** [comment]
Now that LICENSE is included in the tarball, it could be cleaned up a little, set the distfile to be the tarball, no build_style, and `do_install() { vbin ssh-audit.py ; vlicense LICENSE }`

**Comment by RunningDroid** [comment]
Like this?

File: srcpkgs/ssh-audit/template [hunk 1]
```diff
@@ -1,22 +1,18 @@
...
- https://raw.githubusercontent.com/arthepsy/ssh-audit/${_licver}/LICENSE"
-checksum="a3ff0734e1c71b0830a1ff93de2a7bce5faaa8fa893d760966de50eac24d859b
- 8de5f3aaee4a896c62a5c3010954dbe13779b9ae07addcb0380873a9830c66e8"
+homepage="https://github.com/jtesta/ssh-audit"
+distfiles="https://github.com/jtesta/ssh-audit/archive/v${version}.tar.gz"
+checksum=e70d7dfb98fa4941f07424783a2f601c9e3920eb33da53997c13b9d7d2720dcb
+python_version=3
 
 do_install() {
-	vbin ssh-audit.py ssh-audit
```
Comment (abenson) [inline srcpkgs/ssh-audit/template]: Sorry, this could've stayed the same. But yeah, this looks much cleaner!

Comment (RunningDroid) [reply srcpkgs/ssh-audit/template]: fixed

**Comment by abenson** [comment]
Looks great!

**Pull request merged**